tests/conftest_extras_w_cci.py

```diff
@@ -62,7 +63,29 @@ def no_services(*args, **kwargs):
         yield fallback_org_config
 
 
-vcr_config = fixture(cci_vcr_config, scope="module")
+# TODO: Port this back to CCI
+def sf_before_record_response(response):
+    # salesforce_bulk needs the Content-Type header.
+    print(response["headers"])
```

Comment:
Do you want to include this, or is this leftover from debugging?

Reply:
Debugging. I'll take it out.